ScoutSuite/providers/azure/authentication_strategy.py

```diff
@@ -101,55 +64,32 @@ def authenticate(self,
         try:
 
             # Set logging level to error for libraries as otherwise generates a lot of warnings
-            logging.getLogger('adal-python').setLevel(logging.ERROR)
-            logging.getLogger('msrest').setLevel(logging.ERROR)
-            logging.getLogger('msrestazure.azure_active_directory').setLevel(logging.ERROR)
-            logging.getLogger('urllib3').setLevel(logging.ERROR)
-            logging.getLogger('cli.azure.cli.core').setLevel(logging.ERROR)
+            logging.getLogger('azure.identity').setLevel(logging.ERROR)
+            logging.getLogger('azure.core.pipeline').setLevel(logging.ERROR)
 
             context = None
 
             if cli:
-                arm_credentials, subscription_id, tenant_id = \
-                    get_azure_cli_credentials(with_tenant=True)
-                aad_graph_credentials, placeholder_1, placeholder_2 = \
-                    get_azure_cli_credentials(with_tenant=True, resource='https://graph.windows.net')
+                identity_credentials = AzureCliCredential()
 
             elif user_account:
```

Comment:
So it's not possible to authenticate with a MFA token with this mode?

Reply:
Yes it is impossible according to the azure-identity documentation
https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.AzureCliCredential

Reply:
I meant `UsernamePassword`, but you answered this already!